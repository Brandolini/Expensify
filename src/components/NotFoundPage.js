import { Link } from "react-router-dom";
import React from "react";

const NotFoundPage = () => (
  <div>
    404! - Got to <Link to="/">Home</Link>
  </div>
);

export default NotFoundPage;
