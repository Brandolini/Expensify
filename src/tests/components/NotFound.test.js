import NotFoundPage from "../../components/NotFoundPage";
import React from "react";
import { shallow } from "enzyme";

test("should test NotFoundPage", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
