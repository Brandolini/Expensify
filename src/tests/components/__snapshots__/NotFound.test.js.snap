// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test NotFoundPage 1`] = `
<div>
  404! - Got to 
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
